examples/timer/src/main.rs

```diff
@@ -34,13 +39,25 @@ impl Component for Model {
         Self {
             link,
             job: None,
+            clock_job: None,
+            time: "".to_string(),
             messages: Vec::new(),
             _standalone: handle,
         }
     }
 
     fn update(&mut self, msg: Self::Message) -> ShouldRender {
         match msg {
+            Msg::StartClock => {
+                self.time = String::from(js_sys::Date::new_0().to_time_string());
+                let handle = IntervalService::spawn(
+                    Duration::from_secs(1),
+                    self.link.callback(|_| Msg::Tock),
+                );
+                self.clock_job = Some(Box::new(handle));
+
+                true
+            }
```

Comment:
Since there's no way for the user to start or pause the clock manually, why don't you move this to the `create` method?
If you want to add a button to pause the clock that's fine too, but I think that's already covered by the interval ticker